### ISSUE TITLE
Border highlighting of focused windows matching the theme colors.

### ISSUE THREAD

**Issue by the-saddest-clown** [SebastJava/mint-themes#54]
**Is your feature request related to a problem? Please describe.**  
Better visibility of focused windows when using dark themes. See [linuxmint/mint-themes/issues/426](https://github.com/linuxmint/mint-themes/issues/426)

**Describe the solution you'd like**  
Border coloring according to the theme color on the focused window.

**Screenshots**  
![Capture d’écran_2023-01-25_23-40-51](https://user-images.githubusercontent.com/83234199/214708286-a0ec2077-d28d-4b65-8966-8135cdd1326f.png)
This is a Xfwm4 window modified with the #E8602E orange from the Mint-Yz-Dark-Orange70s theme.

**Additional context**  
When having lot of windows open while using a dark theme it can sometimes be difficult to quickly differentiate which one have the focus. Adding a color border to the focused window may resolve the issue.

**Comment by SebastJava** [comment]
Yes. I agree something should be done. Especially on the dark themes. But not forgetting the light (base) themes too. I saw your [https://github.com/linuxmint/mint-themes/issues/426](https://github.com/linuxmint/mint-themes/issues/426) and i saw the other issue you are referring to there.

This thin border works pretty well here, with the sharp contrast between orange and dark gray. But i am not so sure about all the other color combinations. I could try something with a 2 px border, but JosephMcc said somewhere else that he wasn't sure if it could be possible to get a 2px thick border, not just this 1px thickness.

Or it could be something else. Maybe some change on the title bar's background. There should be some more accent on the focused window, somehow, and also, just more contrast between the background and the border, focused or not.

P.S.: I am currently very busy. I hope i will be back working on the GTK themes sometime before April...

**Comment by the-saddest-clown** [comment]
I haven't tested with all the colors but in general the result is pretty nice.
I must admit that I really like the Orange70's with the dark theme.
In my opinion 1px is enough to not be distracting, especially when using bright colors on a dark theme but, yes maybe a 2px border would fit better on a light theme.
Keep up the good work, I really enjoy your theme.
If I were able to tweak the gtk themes I would fork your theme with the border color edition but I'm stuck with gtk editing at this moment. Still trying tho.

**Comment by SebastJava** [comment]
@the-saddest-clown Sorry for not doing any work on this. Here is why: https://github.com/SebastJava/mint-yz-theme/discussions/57

(In there, you will also find some hints about what is possibly coming up next...)
